dev-packages/browser-integration-tests/utils/generatePlugin.ts

```diff
@@ -115,7 +115,7 @@ export const LOADER_CONFIGS: Record<string, { options: Record<string, unknown>;
  * When using compiled versions of the tracing and browser packages, their aliases look for example like
  *     '@sentry/browser': 'path/to/sentry-javascript/packages/browser/esm/index.js'
  * and all other monorepo packages' aliases look for example like
- *     '@sentry/react': 'path/to/sentry-javascript/packages/react'
+ *     '@sentry/hub': 'path/to/sentry-javascript/packages/hub'
```

Comment:
```suggestion
 *     '@sentry/react': 'path/to/sentry-javascript/packages/react'
```

I guess we can leave this as it was?